### PR TITLE
Stop global event listener

### PR DESCRIPTION
**Pull request by x5hemmax**
what do you think?

### CODE REVIEW

**Comment by itoogii** [comment]
Hi Max, you may want to select Agam as reviewer. You can assign that from the right pane by clicking the gear icon for reviewer. I saw you haven't made any changes in this PR other than your empty test file, so I assume it was your test PR to experiment.

**Comment by x5hemmax** [comment]
uh, no that sounds wrong. i will see what agam says

**Comment by x5hemmax** [comment]
thanks for telling me to assign someone, I didn't know that

**Comment by itoogii** [comment]
@x5hemmax I meant to say you need to select @agamm as reviewer.

**Comment by itoogii** [comment]
@agamm I have added the detachGlobalEventListener function and it seems like working in my test. Can you please review and approve the change.

File: src/pages/Content/index.js [hunk 1]
```diff
@@ -38,29 +40,35 @@ function sendMessage(message) {
...
-    }
-  });
+  document.body.addEventListener('click', listener);
+}
+// a function that listens to click events and stores them in the recordedEvents array
+function detachGlobalEventListeners() {
+  // remove the click listener
+  document.body.removeEventListener('click', listener);
+}
+//Function to handle the event listener
```
Comment (agamm) [inline src/pages/Content/index.js:50]:
```suggestion
// Function to handle the global event listener
```

**Review by agamm — APPROVED** [review]
Looks good to me! only change a small comment wording.

**Comment by itoogii** [comment]
Fixed the merge conflicts and tested it working.

**Pull request merged**